webUI/react/src/components/navbar/ReaderNavBar.tsx

```diff
@@ -311,6 +304,22 @@ export default function ReaderNavBar(props: IProps) {
                                     </MenuItem> */}
                                 </Select>
                             </ListItem>
+                            <ListItem>
+                                <ListItemText primary="Reading Direction" />
+                                <Select
+                                    value={settings.readerDirection}
+                                    onChange={(e) => setSettingValue('readerDirection', e.target.value)}
+                                >
+                                    <MenuItem value="LeftToRight">
+                                        Left to right
+
+                                    </MenuItem>
+                                    <MenuItem value="RightToLeft">
+                                        Right to left
+
+                                    </MenuItem>
```

Comment:
You kind of need a matrix to handle all possible cases, maybe put it in an object, 
```
{
SinglePage: ['LeftToRight','RightToLeft','Vertical'],
Vertical: ['Continues', 'Webtoon'],
Horizantal: ['LTR','RTL']
}
```

This naming conversion breaks when you have booth Webtoon and Continues?

Reply:
That'll indeed break the naming convention yeah.

I'm treating the reader types as the how the images are being layout in the reader, and the reading direction as the direction you need to press or click to go towards next page. 

What's your opinion on it?